docs/INSTALL.md

```diff
@@ -468,6 +468,15 @@ If you are low on RAM, disk space or traffic there are ways to reduce system req
 3. Run `./configure.sh` from _Git (for Window) Bash_ and ignore all errors related to Boost.
 4. Go to `./3party/boost`, run `./bootstrap.bat`, and then `b2 headers` to configure Boost.
 
+#### Windows 10 & 11: Enabling symlinks
+
+In order to avoid problems with symlinks while building Android app under Windows, the following steps shall be followed **before** cloning the OM repository:
```

Comment:
Shouldn't this instruction be at the start of this manual?

Reply:
You're right!! I've added these instructions at the beginning of the manual, together with the  instructions to enable git symlinks...